### DISCUSSION TITLE
Replacement for built-in HomeKit?

### DISCUSSION THREAD

**Discussion by tebasham** [d0n4v4nb3ck3r/homebridge-honeywell-home#419] [Q&A] [answered]
Will this work as a reliable replacement for Honeywell’s own buggy implementation of HomeKit? I have a new model T5 that constantly goes “no response” in the Home app and Honeywell is no help at all.

**Comment by donavanbecker** [comment, marked as answer]
@tebasham, yes multiple people have used this plugin for that specific reason.

Just remove the native HomeKit thermostat. Then set this plugin up and if all setup correctly you will be able to control your thermostat with this plugin.
